ios/RCCTabBarController.m

```diff
@@ -166,6 +166,10 @@ - (instancetype)initWithProps:(NSDictionary *)props children:(NSArray *)children
     id selectedIcon = tabItemLayout[@"props"][@"selectedIcon"];
     if (selectedIcon) {
       iconImageSelected = [RCTConvert UIImage:selectedIcon];
+      
+      if (!selectedButtonColor) {
```

Comment:
Sorry to trouble you, Can you please include code samples to help me understand the flow this fixes?